packages/desktop-client/src/components/modals/ImportTransactionsModal/ImportTransactionsModal.jsx

```
@@ -843,13 +849,37 @@
+                  filename,
+                  getParseOptions(filetype, {
+                    delimiter,
+                    hasHeaderRow,
+                    skipLines,
+                    fallbackMissingPayeeToMemo,
+                    importNotes: !importNotes,
+                  }),
+                );
+              }}
+            >
+              <Trans>Import notes from file</Trans>
```

ESLint

[error] 'Trans' is not defined.

(react/jsx-no-undef)